corehq/apps/hqadmin/reports.py

```diff
@@ -1331,3 +1335,80 @@ def rows(self):
                     event.event_date, event.user, event.domain or '', event.ip_address, event.request_path
                 ])
         return rows
+
+
+class UserListReport(GetParamsMixin, AdminReport):
+    base_template = 'reports/base_template.html'
+
+    slug = 'user_list_report'
+    name = ugettext_lazy("User List")
+
+    fields = [
+        'corehq.apps.reports.filters.simple.SimpleSearch',
+    ]
+    emailable = False
+    exportable = False
+    ajax_pagination = True
+    default_rows = 10
+
+    @property
+    def headers(self):
+        return DataTablesHeader(
+            DataTablesColumn(_("Username")),
+            DataTablesColumn(_("Project Spaces")),
+            DataTablesColumn(_("Date Joined")),
+            DataTablesColumn(_("Last Login")),
+            DataTablesColumn(_("Type")),
+            DataTablesColumn(_("SuperUser?")),
+        )
+
+    @property
+    def rows(self):
+        for user in self._get_page(self._users_query()):
+            yield [
+                self._user_link(user['username']),
+                self._get_domains(user),
+                self._format_date(user['date_joined']),
+                self._format_date(user['last_login']),
+                user['doc_type'],
+                user['is_superuser'],
+            ]
+
+    def _users_query(self):
+        query = (user_es.UserES()
+                 .remove_default_filters()
+                 .OR(user_es.web_users(), user_es.mobile_users()))
```

Comment:
why is this necessary? what other kind of users would this filter?

Reply:
there are `UnknownUser`, `AdminUser`, and `demo_user`.  Testing it out locally, I ran into issues because those "users" don't have the full set of fields available, so I figured it'd be easier to explicitly scope this to normal web and mobile users.